### ISSUE TITLE
Issue with "Django Tutorial Part 2: Creating a skeleton websit…": Problem redirecting to /catalog

### ISSUE THREAD

**Issue by Resurgemuss** [mdn/content#6045]
MDN URL: https://developer.mozilla.org/en-US/docs/Learn/Server-side/Django/skeleton_website

#### What information was incorrect, unhelpful, or incomplete?
Dear Mozilla team you are fantastic, thank you so much for the Django tutorial, it is much easier to understand and straightforward than the polls tutorial on Django website. I have an issue though, I have followed exactly and rechecked everything on "part 2: creating a skeleton website" but after navigating http://127.0.0.1:8000/ it does not redirect to error /catalog, unless I write it specifically in the browser. It remains on the rocket  (The install worked successfully!), I have checked every part a couple of times, can you please tell me what is the issue? after  makemigrations it surprised me that it only said "no changes detected" and "migrate" run many migrations and create the sql file.

Thank you so much, everything was going great and I'd like to solve this before continue with the rest of the tutorial, which is really good, thanks again.

#### Did you test this? If so, how?

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/learn/server-side/django/skeleton_website`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/Server-side/Django/skeleton_website
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/server-side/django/skeleton_website/index.html
* Last commit: https://github.com/mdn/content/commit/16f66c643c9aa67a7839925d261f8c2775f0305b
* Document last modified: 2021-05-14T07:57:34.000Z

</details>

**Comment by Resurgemuss** [comment]
I have already solved the issue! :) I decided to install an old version of Django, but because it did give me more issues, I installed again the version I had with the redirecting issue (version 3.2.4) but now it works perfectly! I cannot explain why now it works, honestly.

So it seems that this issue is solved by reinstalling again Django, I am not going to delete this post in case that can be useful for somebody.

Thank you!
